### PR TITLE
Draft June 2024 maintainers minutes

### PR DESCRIPTION
**Pull request by jfy133**
@netlify /blog/2024/maintainers-minutes-2024-06-07

### CODE REVIEW

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+import sad_maintainers_duck from '@assets/images/blog/maintainers-minutes-2024-06-07/sad-maintainers-duck.png';
+import { Image } from 'astro:assets';
+
+In this new initiative, the 'Maintainers Minutes' aims to further give insight into the workings of the [nf-core maintainers team](/governance#maintainers) by providing brief summaries of the monthly Maintainer's team meetings.
+
+## Goodbyes and welcomes!
+
+<Image
+    src={ad_maintainers_duck}
```
Comment (muffato) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:21]:
```suggestion
    src={sad_maintainers_duck}
```

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    class="d-block m-auto"
+    width={400}
+    density={[1.5, 2]}
+    alt="Sad looking cartoon yellow rubber duck with nf-core logo badge on it's body, with a large blue tear coming out of it's eye."
```
Comment (muffato) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:25]:
```suggestion
    alt="Sad looking cartoon yellow rubber duck with nf-core logo badge on its body, with a large blue tear coming out of its eye."
```

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+We once again again spent some time talking about the naming of subworkflows, talking about the tricky balance of keeping findability but conciseness in understanding what a particular subworkflow does.
+
+There was general agreement that actually the _length_ of the subworkflow name isn't as important as ensuring knowing exactly what the subworkflow does.
+In some cases it was argued that describing every operation and tool of the subworkflow was unnecessary and made it harder to understand.
+
+One factor brought up is in complex subworkflows the current '_<tool>` suffix can make subworkflow names very long, and doesn't provide much information compared to active 'verbs'.
+However it was countered that you may have multiple subworkflows that do similar things (e.g. deduplication), so having the tool name performing the activity can be useful.
+
+We agreed as 'homework' for each team member to idependently rename the '[preprocess_rnaseq](https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/preprocess_rnaseq/main.nf)', in their preferred manner, and see if we can find a majority consensus of a naming specification..
```
Comment (muffato) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:57]:
```suggestion
We agreed as 'homework' for each team member to independently rename the '[preprocess_rnaseq](https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/preprocess_rnaseq/main.nf)', in their preferred manner, and see if we can find a majority consensus of a naming specification..
```

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+## Infrastructure Team Updates
+
+Júlia updated us on a couple of developments happening from the infrastructure team.
+
+The version of nf-core tools used for setting up pipelines, and also as to which version of the nf-core template is being used will now be recorded in each pipeline.
+
+A discussion on how to reformat the nf-core modules `meta.yml` file was had, mainly in regards to better displaying metadata associated to input channels with tuples.
+
+Maxime brought up a long running request to have additional fields for better recording publication information (e.g. bibtext information), to potentially allow auto-generation of methods texts with proper attribution of tools.
+
```
Comment (maxulysse) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:80]:
```suggestion
```

**Review by mirpedrol — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please 🥺

**Comment by maxulysse** [comment]
@mirpedrol any ideas why tests are still failing?

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+There was general agreement that actually the _length_ of the subworkflow name isn't as important as ensuring knowing exactly what the subworkflow does.
+In some cases it was argued that describing every operation and tool of the subworkflow was unnecessary and made it harder to understand.
+
+One factor brought up is in complex subworkflows the current '\_<tool>` suffix can make subworkflow names very long, and doesn't provide much information compared to active 'verbs'.
+However it was countered that you may have multiple subworkflows that do similar things (e.g. deduplication), so having the tool name performing the activity can be useful.
+
+We agreed as 'homework' for each team member to independently rename the '[preprocess_rnaseq](https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/preprocess_rnaseq/main.nf)', in their preferred manner, and see if we can find a majority consensus of a naming specification..
+
+If you also want to participate, please let us know of your own ideas!
```
Comment (FriederikeHanssen) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:59]: Where?

Comment (maxulysse) [reply src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:59]: On slack

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+-   Lili Andersson-Lili ([@LilyAnderssonLee](https://github.com/LilyAnderssonLee))
+
+## Still naming things...
+
+We once again again spent some time talking about the naming of subworkflows, talking about the tricky balance of keeping findability but conciseness in understanding what a particular subworkflow does.
+
+There was general agreement that actually the _length_ of the subworkflow name isn't as important as ensuring knowing exactly what the subworkflow does.
+In some cases it was argued that describing every operation and tool of the subworkflow was unnecessary and made it harder to understand.
+
+One factor brought up is in complex subworkflows the current '\_<tool>` suffix can make subworkflow names very long, and doesn't provide much information compared to active 'verbs'.
```
Comment (mashehu) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:54]:
```suggestion
One factor brought up is in complex subworkflows the current '\_$tool` suffix can make subworkflow names very long, and doesn't provide much information compared to active 'verbs'.
```
To not break the build.

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
Checks pass now

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+There was general agreement that actually the _length_ of the subworkflow name isn't as important as ensuring knowing exactly what the subworkflow does.
+In some cases it was argued that describing every operation and tool of the subworkflow was unnecessary and made it harder to understand.
+
+One factor brought up is in complex subworkflows the current '\_$tool` suffix can make subworkflow names very long, and doesn't provide much information compared to active 'verbs'.
+However it was countered that you may have multiple subworkflows that do similar things (e.g. deduplication), so having the tool name performing the activity can be useful.
+
+We agreed as 'homework' for each team member to independently rename the '[preprocess_rnaseq](https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/preprocess_rnaseq/main.nf)', in their preferred manner, and see if we can find a majority consensus of a naming specification..
+
+If you also want to participate, please let us know of your own ideas!
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:60]:
```suggestion
If you also want to participate, please let us know of your own ideas on the [#subworkflow](https://nfcore.slack.com/archives/C03D38JNFNJ) channel on the nf-core slack!
```

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx (hunk 5 above)
Comment (jfy133) [reply src/content/blog/2024/maintainers-minutes-2024-06-07.mdx]: Updated

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 8]
```diff
@@ -64,17 +64,17 @@ If you also want to participate, please let us know of your own ideas on the [#s
...
-To improve the experience of proposers by increasing turnaround time, and reducing the load on the core team on evaluating proposals outside their 'comfort zone', the maintainers team has agreed to help monitor the #new-pipelines-proposal channel and evaluate each new submission.
+To improve the experience of proposers by decreasing turnaround time, and reducing the load on the core team on evaluating proposals outside their 'comfort zone', the maintainers team has agreed to help monitor the #new-pipelines-proposal channel and evaluate each new submission.
 From now one each proposal can have one core-team member and one maintainer approval for a pipeline to be accepted!
 
 ## Infrastructure Team Updates
 
 Júlia updated us on a couple of developments happening from the infrastructure team.
 
-The version of nf-core tools used for setting up pipelines, and also as to which version of the nf-core template is being used will now be recorded in each pipeline.
+In the upcoming version of the nf-core template, the version of nf-core tools used for setting up a pipeline, and also which version of the nf-core template is currently being used will now be recorded in each pipeline.
```
Comment (JoseEspinosa) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:77]:
```suggestion
In the upcoming version of the nf-core template, the version of nf-core tools used for setting up a pipeline, and also which version of the nf-core template is currently being used will be recorded in each pipeline.
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! 🚀 
Just a small suggestion

File: src/content/blog/2024/maintainers-minutes-2024-06-07.mdx [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+## Infrastructure Team Updates
+
+Júlia updated us on a couple of developments happening from the infrastructure team.
+
+In the upcoming version of the nf-core template, the version of nf-core tools used for setting up a pipeline, and also which version of the nf-core template is currently being used will be recorded in each pipeline.
+
+A discussion on how to reformat the nf-core modules `meta.yml` file was had, mainly in regards to better displaying metadata associated to input channels with tuples.
+
+Maxime brought up a long running request to have additional fields for better recording publication information (e.g. bibtext information), to potentially allow auto-generation of methods texts with proper attribution of tools.
```
Comment (maxulysse) [inline src/content/blog/2024/maintainers-minutes-2024-06-07.mdx:81]:
```suggestion
Maxime brought up a long running request to have additional fields for better recording publication information (e.g. bibtex information), to potentially allow auto-generation of methods texts with proper attribution of tools.
```

**Review by maxulysse — APPROVED** [review]

**Pull request merged**